tests/pipeline/test_blaze.py

```diff
@@ -843,10 +849,21 @@ def compute(self, today, assets, out, data):
             check_dtype=False,
         )
 
-    @with_extra_sid
-    def test_deltas(self, asset_info):
-        expr = bz.Data(self.df, name='expr', dshape=self.dshape)
-        deltas = bz.Data(self.df, dshape=self.dshape)
+    @with_ignore_sid
+    def test_deltas(self, asset_info, extra_sid_to_ignore):
```

Comment:
`extra_sid_to_ignore` sounds like it should the `sid` itself, maybe we could rename to something like `add_extra_sid` to show that it is actually a boolean. What do you think?